cli/provisionerjobs.go

```diff
@@ -41,7 +41,7 @@ func (r *RootCmd) provisionerJobsList() *serpent.Command {
 		client     = new(codersdk.Client)
 		orgContext = NewOrganizationContext()
 		formatter  = cliui.NewOutputFormatter(
-			cliui.TableFormat([]provisionerJobRow{}, []string{"created at", "id", "organization", "status", "type", "queue", "tags"}),
+			cliui.TableFormat([]provisionerJobRow{}, []string{"id", "created at", "status", "tags", "type", "organization", "queue"}),
```

Comment:
```suggestion
			cliui.TableFormat([]provisionerJobRow{}, []string{"created at", "id", "type", "template display name", "status", "queue", "tags"}),
```
I figure you changed this to reduce the diff, but while we're at it we can make it a bit better. The changes here are inspired by the design work (Figma) but also adapted to CLI needs.